build-system/tasks/prettify.js

```diff
@@ -24,26 +24,22 @@
 
 const argv = require('minimist')(process.argv.slice(2));
 const globby = require('globby');
+const gulp = require('gulp');
 const log = require('fancy-log');
+const path = require('path');
+const prettier = require('gulp-prettier');
```

Comment:
Which version of prettier gets used with `gulp-prettier`? Ours or theirs?

Reply:
Ours. See response below.